test/unit/search/kmer_index/shape_test.cpp

```diff
@@ -0,0 +1,55 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2019, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2019, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE.md
+// -----------------------------------------------------------------------------------------------------
+
+#include <gtest/gtest.h>
+
+#include <seqan3/search/kmer_index/shape.hpp>
+#include <seqan3/std/ranges>
+
+using namespace seqan3;
+
+// ============================================================================
+//  shape
+// ============================================================================
+
+TEST(shape, ctr)
+{
+    EXPECT_TRUE((std::is_default_constructible_v<shape>));
+    EXPECT_TRUE((std::is_nothrow_default_constructible_v<shape>));
+    EXPECT_TRUE((std::is_copy_constructible_v<shape>));
+    EXPECT_TRUE((std::is_trivially_copy_constructible_v<shape>));
+    EXPECT_TRUE((std::is_nothrow_copy_constructible_v<shape>));
+    EXPECT_TRUE((std::is_move_constructible_v<shape>));
+    EXPECT_TRUE((std::is_trivially_move_constructible_v<shape>));
+    EXPECT_TRUE((std::is_nothrow_move_constructible_v<shape>));
+    EXPECT_TRUE((std::is_copy_assignable_v<shape>));
+    EXPECT_TRUE((std::is_trivially_copy_assignable_v<shape>));
+    EXPECT_TRUE((std::is_nothrow_copy_assignable_v<shape>));
+    EXPECT_TRUE((std::is_move_assignable_v<shape>));
+    EXPECT_TRUE((std::is_trivially_move_assignable_v<shape>));
+    EXPECT_TRUE((std::is_nothrow_move_assignable_v<shape>));
+
+    shape s0(0, 0, 0);
```

Comment:
```suggestion
    shape s0{0, 0, 0};
```
also everywhere else